plugins/UM3NetworkPrinting/ClusterUM3OutputDevice.py

```diff
@@ -479,7 +482,7 @@ def _updatePrinter(self, printer: PrinterOutputModel, data: Dict) -> None:
         printer.updateKey(data["uuid"])
         printer.updateType(data["machine_variant"])
 
-        # Do not store the buildplate information that comes from connect if the current printer has not buildplate information
+        # Do not store the build plate information that comes from connect if the current printer has not build plate information
         if "build_plate" in data and machine_definition.getMetaDataEntry("has_variant_buildplates", False):
             printer.updateBuildplateName(data["build_plate"]["type"])
```

Comment:
@Ghostkeeper Should we be updating all cases of single-word buildplate with two-word build plate?